src/CodeCompliance/Domain/Checks/Filters/AbstractIgnoreListFilter.php

```diff
@@ -50,6 +50,9 @@ public function getIgnoreSuffixes(): array
             '/\/Development\//',
             '/^Spryker\/Zed\/\w+DataImport\//',
             '/^Spryker\/Shared\/Twig\/TwigFunctionProvider$/',
+            '/^Spryker\/Zed\/Gui\/Communication\/Table\/AbstractTable$/',
+            '/^Spryker\/Zed\/Gui\/Communication\/Tabs\/AbstractTabs$/',
+            '/^Spryker\/DecimalObject\//',
```

Comment:
public methods should be placed above protected ones

Reply:
The method was changed to protected